src/systemctl/systemctl-enable.c

```diff
@@ -61,6 +61,33 @@ static int normalize_names(char **names) {
         return 0;
 }
 
+static void warn_triggering_units_for_masked(sd_bus *bus, const char *unit) {
```

Comment:
this is almost identical to `warn_triggering_units_for_stopped()`, apart from the ignore_masked bool and the first word of the warning, so maybe just merge them and add a bool parameter?